src/systemctl/systemctl.c

```diff
@@ -5666,6 +5711,12 @@ static int enable_unit(int argc, char *argv[], void *userdata) {
                 const char *method;
                 sd_bus *bus;
 
+                if (STR_IN_SET(verb, "mask", "unmask")) {
+                        r = unit_exists(*names);
+                        if (r < 0)
+                                return r;
```

Comment:
with the suggested change this would then become:

```c
…
r = unit_exists(*names);
if (r < 0)
        return r;
if (r == 0)
        log_notice("Unit %s does not exist, proceeding anyway.", *names);
```